github-orgs/kubeflow/org.yaml

```diff
@@ -155,6 +155,7 @@ orgs:
         - yupbank
         - ziamsyed
         - zjj2wry
+        - jinchihe
```

Comment:
@jinchihe please follow alpha order in this list.

Reply:
@gyliu513 Updated, thanks.